packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -240,8 +240,13 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     // For split buttons, touching anywhere in the button should drop the dropdown, which should contain the primary action.
     // This gives more hit target space for touch environments. We're setting the onpointerdown here, because React
     // does not support Pointer events yet.
```

Comment:
i don't feel these comments are relevant. i think the code is pretty self-explain here. pls remove them if you agree

Reply:
@xugao we've had issues before with people complaining about this behavior, which is why the comment was added in the first place. With subsequent issues we've just pointed them to this comment, so I wouldn't feel completely comfortable with removing it.